test/functional/fixtures/api/es-next/take-screenshot/test.js

```diff
@@ -292,6 +307,21 @@ describe('[API] t.takeElementScreenshot()', function () {
                 });
         });
 
+        it('Should check the path argument for forbidden characters', function () {
+            return runTests('./testcafe-fixtures/take-element-screenshot.js', 'Forbidden characters in the path argument', {
+                shouldFail: true,
+                only:       'chrome'
+            })
+                .catch(function (errs) {
+                    expect(errs[0]).to.contains('There are forbidden characters in the "path:with*forbidden|chars" screenshot path: ":" at index 5 "*" at index 10 "|" at index 20');
```

Comment:
We don't have any separator between two forbidden symbols' description. It could be hard to read, what do you think?

Reply:
It's not a real message, all whitespace symbols in the expected value (including `\n` and `\t`) are replaced by spaces. Look here for the real message: https://github.com/DevExpress/testcafe/pull/2820/files#diff-7da3eea11b9cded2babff4ae9262a7ddR1

Each forbidden symbol is displayed on a dedicated line with indentation.